packages/insomnia-app/app/common/documentation.ts

```diff
@@ -1,4 +1,4 @@
-const insomniaDocs = (slug: string) => `https://support.insomnia.rest${slug}`;
+const insomniaDocs = (slug: string) => `https://docs.insomnia.rest${slug}`;
 
 export const docsBase = insomniaDocs('/');
 export const docsGitSync = insomniaDocs('/article/193-git-sync');
```

Comment:
We should also update the docs links to be correct. The redirects are defined [here](https://github.com/Kong/insomnia-docs/blob/main/docs/vercel.json). This line for example, would become

```suggestion
export const docsGitSync = insomniaDocs('/insomnia/git-sync');
```

Reply:
good catch!  @falondarville could you go through them? if not lemme know

Reply:
Yep, I can pull this work and go through all of these!

Reply:
@dimitropoulos permission denied to push to your fork. Adding the fixes here til we can resolve permission issues:

```
export const docsBase = insomniaDocs('/');
export const docsGitSync = insomniaDocs('/insomnia/git-sync');
export const docsTemplateTags = insomniaDocs('/insomnia/template-tags');
export const docsVersionControl = insomniaDocs('/insomnia/version-control-sync');
export const docsPlugins = insomniaDocs('/insomnia/introduction-to-plugins');
export const docsImportExport = insomniaDocs('/insomnia/import-export-data');
export const docsKeyMaps = insomniaDocs('/insomnia/key-maps');
export const docsIntroductionInsomnia = insomniaDocs('/insomnia/get-started');
export const docsWorkingWithDesignDocs = insomniaDocs('/insomnia/design-documents');
```

Reply:
(update: got it sorted out)